### ISSUE TITLE
Make an RFC for accessibility in new syntax

### ISSUE THREAD

**Issue by spdegabrielle** [racket/racket2-rfcs#18]
I’m suggesting accessibility sight impaired and blind developers should be explicitly on the agenda
